packages/react/src/components/footer/footerUtilities/FooterUtilities.test.tsx

```diff
@@ -1,54 +1,46 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import { axe } from 'jest-axe';
 
 import { FooterUtilities } from './FooterUtilities';
+import { FooterNavigationLink } from '../footerNavigationLink/FooterNavigationLink';
 import { FooterWrapper } from '../../../utils/test-utils';
 
 describe('<Footer.Utilities /> spec', () => {
   it('renders the component', () => {
-    const { asFragment } = render(<FooterUtilities />, { wrapper: FooterWrapper });
+    const { asFragment } = render(
+      <FooterUtilities>
```

Comment:
On XS and S screen sizes, Footer utility links should stack on top of each other OR at least have --spacing-m gaps. Smaller gaps affect readability, sometimes they might look like regular spaces.

Reply:
Good find! That's just something that slipped my attention 😄 I made them stack up, you can check the demo.